packages/cli/src/commands/__tests__/deploy.test.ts

```diff
@@ -23,7 +30,7 @@ describe('deploy', () => {
 
   describe('skipping prompts', () => {
     it('should exit if scope is not provided', async () => {
-      await expect(() =>
+      await expect(
```

Comment:
Don't you need `() =>` here? From my memory, when you're testing asynchronous functions like this, you have to pass a function to `expect()`, not a function call 🤔

Reply:
yeah I know what you mean but I tested it and it worked without it too. Can you maybe also check on your computer, maybe it is because of the vitest version update.

Reply:
Ah, it worked on my mac. 

I guess both ways are supported!

![Screenshot 2023-08-10 at 11 58 50](https://github.com/storyblok/field-plugin/assets/499898/961cbc1a-ab8d-4ae7-8611-4355f13336a9)

https://vitest.dev/api/expect.html#resolves